### PR TITLE
Completed Issue 2 user can view entries - Jesus Araujo

### PR DESCRIPTION
**Pull request by j-alejandro-araujo**
Screenshot 2023-05-31 at 8 19 24 PM

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -17,12 +18,99 @@ $form.addEventListener('submit', event => {
...
+    var $oldView = document.querySelector('div[data-view="entry-form"]');
+    $oldView.setAttribute('class', 'hidden');
+  } else {
+    $view = document.querySelector('div[data-view="entry-form"]');
+    $view.setAttribute('class', '');
+    data.view = view;
+    $oldView = document.querySelector('div[data-view="entries"');
+    $oldView.setAttribute('class', 'hidden');
+  }
+}
```
Comment (shawnkost) [inline js/main.js]: 2 things

1. Use `const`
2. Move the `querySelector`s outside the function. By having them inside of here, we're having to traverse the DOM every single time this function gets called. If we move these outside of the function into the global space, we can just query them once and always have a reference to them

File: js/main.js [hunk 2]
```diff
@@ -17,12 +18,99 @@ $form.addEventListener('submit', event => {
...
+
+const $entriesAnchor = document.querySelector('#entries-link');
+
+$entriesAnchor.addEventListener('click', event => {
+  event.preventDefault();
```
Comment (shawnkost) [inline js/main.js]: Not needed

File: js/main.js [hunk 3]
```diff
@@ -17,12 +18,99 @@ $form.addEventListener('submit', event => {
...
+  const viewTarget = $entriesAnchor.getAttribute('data-view');
+  viewSwap(viewTarget);
+});
+
+const $entriesFormAnchor = document.querySelector('#new-entry');
+
+$entriesFormAnchor.addEventListener('click', event => {
+  event.preventDefault();
```
Comment (shawnkost) [inline js/main.js]: Not needed

**Review by shawnkost — CHANGES_REQUESTED** [review]
Nicely done Jesus! A few changes here

File: js/main.js (hunk 3 above)
Comment (j-alejandro-araujo) [reply js/main.js]: Adjustments are complete! 
Screenshot 2023-06-01 at 9 59 51 AM

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**